index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="project-description" content="Portfolio">
+  <meta name="keywords" content="HTML, CSS">
+  <meta name="author" content="Lucy W. Mwangi">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" rel="stylesheet" />
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <!----------nav bar-->
+  <header>
+    <nav>
+      <a>My Logo</a>
+      <img src="Assets/images/Union.png" alt="">
+    </nav>
+  </header>
+
+  <!-------main section------>
+  <section class="main">
+    <div class="intro-text">
+      <h1>I’m Lucy. <br> Glad to see <br> you!</h1>
+      <p class="text-description">I’m a software developer! I can help you build a product, feature or website Look
+        through some of my work and experience! If you like what you see and have a project you need coded, don’t
+        hestiate to contact me.</p>
+    </div>
+    <div class="social">
+      <h3>LET'S CONNECT</h3>
+      <ul class="media-icons">
+        <li><a><img src="Assets/images/Twitter.png" alt="twitter"></a></li>
+        <li><a><img src="Assets/images/Linkedin icon.png" alt="linkedin"></a></li>
+        <li><a><img src="Assets/images/Mail.png" alt="mail"></a></li>
+        <li><a><img src="Assets/images/Github.png" alt="github"></a></li>
+        <li><a><img src="Assets/images/peace.png" alt="vector"></a></li>
+      </ul>
+    </div>
+  </section>
+ <!-------work section------>
+ <section class="container-work">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give correct style to `list item texts`(`"Canopy"`) and add some space between list of language texts(`<ul class="language-categories">`) and submit `button` 🙏🏼 

![image](https://github.com/lucy-sees/Webforlio/assets/63915024/0f1348ce-6464-4641-baab-9715769b1076)